Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -10,23 +10,28 @@ import UIKit
 import SafariServices
 import Crashlytics
 
-class NewsTableViewController: UITableViewController, NewsCellViewModelDelegate {
-
+class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b3221f0a091d78ec4f94454aba4742e808f6b14d/Headlines/src/Controllers/NewsTableViewController.swift#L13"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 232 lines</td>
    </tr>
  </tbody>
</table>